SI/Sem.04/Pract.04/Task02.cpp

```diff
@@ -0,0 +1,23 @@
+#include <iostream>
+using namespace std;
+
+bool isEven(int num)
+{
+	bool isEven;
+	if (num % 2 == 0)
```

Comment:
num % 2 is a boolean expression, so it returns a boolean value that you can directly return